### PR TITLE
XZY instead of XYZ

### PR DESCRIPTION
**Pull request by ghost**
What do you think about it?

### CODE REVIEW

File: wot/ModelReader.py [hunk 1]
```diff
@@ -320,7 +322,7 @@ def readNormal(self, data, IS_NEW):
 				z = -float(pkz&0x7f)/0x7f
 			else:
 				z = float(pkz^0x7f)/0x7f
-			return (x, y, z)
+			return (x, z, y)
```
Comment (SkaceKamen) [inline wot/ModelReader.py:334]: Are you sure about this? I'm asking because it seemed like it works in this order for some models.

Comment (ghost) [inline wot/ModelReader.py:334]: > Are you sure about this?

I'm sure.

**Comment by ShadowHunterRUS** [comment]
What do you think about last commit?

**Comment by SkaceKamen** [comment]
Sry, I completely forgot. It's all fine, I'm going to trust you :+1:

**Pull request merged**